app/models/prescription.rb

```diff
@@ -53,8 +53,8 @@ class Prescription < Common::Base
   attribute :prescription_name, String, sortable: { order: 'ASC', default: true }
   attribute :dispensed_date, Common::UTCTime, sortable: { order: 'DESC' }
   attribute :station_number, String
-  attribute :is_refillable, Boolean
-  attribute :is_trackable, Boolean
+  attribute :is_refillable, Boolean, filterable: 'eq'
```

Comment:
It doesn't seem like there's a business need for "not equals", but I'd probably add it anyhow just to keep it consistent with the others. It doesn't sound like the `non-refillable` tab is going to happen but that would make it easy for us to implement if we ever needed to.